.github/workflows/windows.yml

```diff
@@ -73,10 +73,13 @@ jobs:
           choco install wixtoolset
           echo "C:\\Program Files (x86)\\WiX Toolset v3.11\\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
       - name: Bootstrap
+        env:
+          RUSTC: "rustc"
```

Comment:
Why is this necessary?

Reply:
I think this should probably have a comment explaining this. It's a bit confusing otherwise. Thanks!